.circleci/config.yml

```diff
@@ -83,14 +83,14 @@ commands:
     steps:
       - macos/preboot-simulator:
           platform: "iOS"
-          version: "16.1"
+          version: "17.0"
           device: "iPhone 14"
```

Comment:
are we updating codecov to 3.3.0 for Edge?

Reply:
I am seeing some issues with the codecov org and this macos/xcode version, and I am trying to confirm if these is a workaround with the Codecov team. we'll take any changes regarding codecov in a separate PR.